packages/styles/components/Input.module.scss

```diff
@@ -0,0 +1,40 @@
+@import '~@venice/styles/core';
+
+.container {
+  display: flex;
+  flex-flow: column;
+  .label {
```

Comment:
You don't need to encapsulate things. All classes here will be generate a CSS Module.

Please using this like a:

```css
.container {}

.label {}
```

and etc...